### PR TITLE
SSUT-225: Introduce packages to scaffolds

### PR DESCRIPTION
**Pull request by TheNickWilson**
Each scaffold will now ask for a package as well as a class name, and place the files in those packages.

### CODE REVIEW

**Comment by giftig** [comment]
I've assigned you SSUT-225 since it's basically what you've done here. Might want to stick that ticket number in your commit title.

File: .g8/checkboxPage/app/views/$package$/$className$View.scala.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import controllers.$package$.{routes => $package$Routes}
```
Comment (giftig) [inline .g8/checkboxPage/app/views/$package$/$className$View.scala.html:1]: the view is within that same package so I think including the package name here is unnecessary; inside `foo.views.BarView` it should be natural to assume `routes` means `controllers.foo.routes`

**Comment by giftig** [comment]
re: adding new packages, can you include the same note you mentioned in slack in the readme? I added a section about using the scaffolds so having a quick note about that there would be a helpful reminder.

File: .g8/checkboxPage/app/views/$package$/$className$View.scala.html (hunk 1 above)
Comment (TheNickWilson) [reply .g8/checkboxPage/app/views/$package$/$className$View.scala.html:1]: Fair enough, I'll amend that :)

**Pull request merged**